lib/features/add_property/view/widgets/add_rooms_view.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:hatspace/features/add_property/view_model/add_property_cubit.dart';
+import 'package:hatspace/strings/l10n.dart';
+
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+class AddRoomsView extends StatefulWidget {
```

Comment:
```suggestion
class AddPropertyRoomsView extends StatefulWidget {
```
For consistency with AddPropertyImagesView and AddPropertyFeaturesView